app/proguard-rules.pro

```diff
@@ -27,6 +27,10 @@
 -keep class org.webrtc.** {*;}
 -keepclasseswithmembernames class * { native <methods>; }
 
+# WebRTC SDP and ICE candidate auxiliary classes serialized into json.
+-keepattributes InnerClasses
+-keep class co.tinode.tindroid.CallFragment$* { *; }
```

Comment:
Why not just `co.tinode.tindroid.CallFragment$IceCandidateAux`? Otherwise you are keeping all inner classed inside this class.

Reply:
BTW, alternatively you can make it Serializable

Reply:
Made both classes Serializable. Thanks.